src/objectExplorer/nodes/treeNodeInfo.ts

```diff
@@ -66,6 +68,13 @@ export class TreeNodeInfo extends vscode.TreeItem implements ITreeNodeInfo {
         } else {
             this.iconPath = ObjectExplorerUtils.iconPath(this.nodeType);
         }
+        if (connectionProfile && connectionProfile.containerName) {
```

Comment:
Since this only applied to connection nodes, this should be present in ConnectionNode class